frontend/src/scenes/persons-management/PersonsManagementScene.test.tsx

```diff
@@ -0,0 +1,127 @@
+                        }
+                        return res(
+                            ctx.json({
+                                results: [testExampleResult],
+                                count: 1,
+                                complete: true,
+                            })
+                        )
+                    }
+                    // if (payload.query.kind !== 'ActorsQuery') {
+                    //     return res(ctx.json({
+                    //         results: [], count: 0,
+                    //         complete: true,
+                    //     }))
+                    // }
+                    // by default we have more than one actor
+
+                    return res(
+                        ctx.json({
+                            results: [
+                                testExampleResult,
+                                [
+                                    {
+                                        id: 2,
+                                        name: 'And another test User',
+                                        distinct_ids: ['test_i2'],
+                                        properties: {
+                                            email: 'test2@example.com',
+                                        },
+                                        created_at: '2021-01-01T00:00:00Z',
+                                    },
+                                ],
+                            ],
+                            count: 1,
+                            complete: true,
+                        })
+                    )
+                },
+            },
+        })
+        initKeaTests()
+        router.actions.push(urls.persons())
+    })
+
+    it('can search for persons', async () => {
+        render(
+            <Provider>
+                <PersonsManagementScene />
+            </Provider>
+        )
+
+        const searchInput = screen.getByPlaceholderText('Search for persons')
+        userEvent.type(searchInput, 'test@example.com')
+
+        const lemonTable = screen.getByTestId('persons-table')
+        const withinTable = within(lemonTable)
+
+        await waitFor(() => {
+            expect(
+                withinTable.getByTitle('This is the Gravatar for test@example.com <test@example.com>')
+            ).toBeInTheDocument()
+            expect(withinTable.getByText('T', { selector: '.Lettermark' })).toBeInTheDocument()
+        })
+    })
+
+    it('can get the empty state for persons', async () => {
+        render(
+            <Provider>
+                <PersonsManagementScene />
+            </Provider>
+        )
+
+        const searchInput = screen.getByPlaceholderText('Search for persons')
+        userEvent.type(searchInput, 'not-in-mock@example.com')
+
+        const lemonTable = screen.getByTestId('persons-table')
+        const withinTable = within(lemonTable)
+
+        await waitFor(() => {
+            // TODO: this shouldn't say events, right?
```

Comment:
Looks like it does at the moment 🤷 

![CleanShot 2025-03-04 at 03 57 36@2x](https://github.com/user-attachments/assets/003b4ac6-a075-4447-9e75-e06962eb8f47)

Reply:
yep, which i guess we "broke" at some point in the past and didn't spot 🙈